fix(theme-translations): complete and improve Spanish theme translations

4troDev opened the pull request:
I've fixed and updated Spanish translations in `theme-common.json` at `packages/docusaurus-theme-translations/locales/es/`. P

## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [x] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved my working plan.

4troDev committed:
Updated es/theme-common.json

Updated Spanish Translations in theme-common.json

slorber commented on packages/docusaurus-theme-translations/locales/es/theme-common.json:
```diff
@@ -3,18 +3,18 @@
   "theme.BackToTopButton.buttonAriaLabel": "Volver al principio",
   "theme.CodeBlock.copied": "Copiado",
   "theme.CodeBlock.copy": "Copiar",
-  "theme.CodeBlock.copyButtonAriaLabel": "Copiar código al portapapeles",
```
I don't mind this change, but the original string is:

>   "theme.CodeBlock.copyButtonAriaLabel": "Copy code to clipboard",

This is an aria label for accessibility. Are you sure being less specific will improve accessibility?

4troDev replied:
I've made a small improvement by replacing `Copiar código al portapapeles `with `Copiar código`. This change maintains clarity without affecting accessibility and improves the text for users on different operating systems.

slorber commented on packages/docusaurus-theme-translations/locales/es/theme-common.json:
```diff
@@ -51,7 +51,7 @@
   "theme.docs.sidebar.collapseButtonTitle": "Colapsar barra lateral",
   "theme.docs.sidebar.expandButtonAriaLabel": "Expandir barra lateral",
   "theme.docs.sidebar.expandButtonTitle": "Expandir barra lateral",
-  "theme.docs.sidebar.navAriaLabel": "Docs barra lateral",
```
Similarly this is an aria label for accessibility

I'm not an expert, but with this change, are you sure a blind person understands this sidebar is related to the docs?

4troDev replied:
thanks for the information i'll updated it based on that

4troDev committed:
Update theme-common.json

Updated the code to enhance specificity. From `Barra lateral`  to `Barra lateral de Documentos` using `Barra lateral ' can be used for both docs and blogs.

4troDev reviewed (commented):
Replaced `Barra lateral` with `Barra lateral de Documentos` for accessibility improvement. However, note that using the "Barra lateral" can work to indicate the sidebar for both docs and blogs.

slorber commented on packages/docusaurus-theme-translations/locales/es/theme-common.json:
```diff
@@ -3,18 +3,18 @@
   "theme.CodeBlock.wordWrapToggle": "Alternar ajuste de palabras",
-  "theme.DocSidebarItem.collapseCategoryAriaLabel": "Collapse sidebar category '{label}'",
-  "theme.DocSidebarItem.expandCategoryAriaLabel": "Expand sidebar category '{label}'",
+  "theme.DocSidebarItem.collapseCategoryAriaLabel": "Colapsar categoría de barra lateral '{label}'",
+  "theme.DocSidebarItem.expandCategoryAriaLabel": "Ampliar la categoría de la barra lateral '{label}'",
```
I'm not Spanish but that doesn't look good to me. `{label}` is the name of the category, not the name of the sidebar. Wouldn't it be better to put this placeholder closer to the "categoria" word?

slorber replied:
Not sure it applies to Spanish, but what we have in French:

```
  "theme.DocSidebarItem.collapseCategoryAriaLabel": "Réduire la catégorie '{label}' de la barre latérale",
  "theme.DocSidebarItem.expandCategoryAriaLabel": "Développer la catégorie '{label}' de la barre latérale",
```

4troDev replied:
{Label} is a place holder if i change it when there is a Label it won't show up since is not in the code

slorber committed:
Update packages/docusaurus-theme-translations/locales/es/theme-common.json
